.changeset/fuzzy-pans-share.md

```diff
@@ -0,0 +1,9 @@
+---
+"@rocket.chat/meteor": minor
+"@rocket.chat/apps": minor
+"@rocket.chat/core-typings": minor
+"@rocket.chat/model-typings": minor
```

Comment:
```suggestion
"@rocket.chat/meteor": major
"@rocket.chat/apps": major
"@rocket.chat/core-typings": minor
"@rocket.chat/model-typings": minor
```
Some changes should be tagged as major now, shouldn't they? Not sure about core and model typings